packages/terra-worklist-data-grid/src/subcomponents/Row.jsx

```diff
@@ -159,7 +159,10 @@ function Row(props) {
 
   return (
     <tr
-      className={cx([isSelected ? 'row-selected' : 'worklist-data-grid-row', theme.className])}
+      className={cx('worklist-data-grid-row', {
+        'row-selected': isSelected,
```

Comment:
I would make this class "selected" instead of "row-selected" to match the pattern for the Cell component.  This is already a Row component so we don't really need to prefix with "row-" with class mangling in place.